packages/internals/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -725,70 +725,20 @@ describe('getGenerators', () => {
       },
     }
 
-    try {
-      await getGenerators({
-        schemaPath: path.join(__dirname, 'missing-models-sqlite-schema.prisma'),
-        providerAliases: aliases,
-      })
-    } catch (e) {
-      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
-        "
-        You don't have any models defined in your schema.prisma, so nothing will be generated.
-        You can define a model like this:
-
-        model User {
-          id    Int     @id @default(autoincrement())
-          email String  @unique
-          name  String?
-        }
-
-        More information in our documentation:
-        https://pris.ly/d/prisma-schema
-        "
-      `)
-    }
-
-    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-  })
-
-  test('fail if no model(s) found - mongodb', async () => {
-    expect.assertions(5)
-    const aliases = {
-      'predefined-generator': {
-        generatorPath: generatorPath,
-        outputPath: __dirname,
-      },
-    }
-
-    try {
-      await getGenerators({
-        schemaPath: path.join(__dirname, 'missing-models-mongodb-schema.prisma'),
-        providerAliases: aliases,
-      })
-    } catch (e) {
-      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
-        "
-        You don't have any models defined in your schema.prisma, so nothing will be generated.
-        You can define a model like this:
-
-        model User {
-          id    String  @id @default(auto()) @map("_id") @db.ObjectId
-          email String  @unique
-          name  String?
-        }
+    const generators = await getGenerators({
+      schemaPath: path.join(__dirname, 'missing-models-sqlite-schema.prisma'),
+      providerAliases: aliases,
+    })
 
-        More information in our documentation:
-        https://pris.ly/d/prisma-schema
-        "
-      `)
-    }
+    expect(generators.length).toEqual(1)
 
     expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
+    expect(stripAnsi(ctx.mocked['console.warn'].mock.calls.join('\n'))).toMatchInlineSnapshot(`
+    "
+    warn You don't have any models defined in your schema.prisma, so nothing will be generated.
```

Comment:
Isn't this what you wanted to change?

To me this message means "generate does not work".
But I think now generate does work, but it will just be an empty Client?

How about keeping the old message about a simple copy-pasteable model name, and instead describe clearer what is happening now?

Reply:
> But I think now generate does work, but it will just be an empty Client?
Yeah, you're right.



> How about keeping the old message about a simple copy-pasteable model name, and instead describe clearer what is happening now?

I didn't really understand what you were trying to suggest here, except to make it clearer that everything works.
So, you're suggesting to leave the last message, where it was stated that you can specify the model and also say that there are no models?

```
You don't have any models defined in your 'schema.prisma', so PrismaClient will be empty.
You can define a model like this:

model User {
  id    String     @id @default(auto()) @map("_id") @db.ObjectId
  email String  @unique
  name  String?
},

More information in our documentation:
${link('https://pris.ly/d/prisma-schema')}
```

Something like this?

Reply:
Yes, exactly. Otherwise that helpful output goes away.

Reply:
I'm pushing change, you'll see when you have time. 🙃